1.10.15/buster/Dockerfile

```diff
@@ -132,6 +132,9 @@ RUN pip install "${AIRFLOW_MODULE}" --constraint /tmp/build-time-pip-constraints
     && pip install "https://github.com/astronomer/astronomer-airflow-scripts/releases/download/v0.0.5/astronomer_airflow_scripts-0.0.5-py3-none-any.whl" \
     && pip install "astronomer-fab-security-manager==${ASTRONOMER_FAB_SECURITY_MANAGER_VERSION}"
 
+LABEL io.astronomer.docker.airflow.version="${AIRFLOW_VERSION}"
```

Comment:
I recall we had found a bug, there might be an issue somewhere where the labels weren’t preserved in a multi stage build

Reply:
Ah, must be fixed now then.

Reply:
Might be https://stackoverflow.com/questions/65331437/difference-in-how-docker-labels-are-applied-in-multi-stage-builds-across-platfor but not sure